djangocms_versioning_filer/monkeypatch/models.py

```diff
@@ -30,6 +48,31 @@ def is_file_content_valid_for_revert(version, user):
         )
 
 
+def model_save(func):
+    def inner(self, *args, **kwargs):
+        func(self, *args, **kwargs)
+        if self.__class__ == File:
```

Comment:
I don't understand how this is supposed to work. This code won't be ran because the original function will have completed by this line.